### PR TITLE
Adaptive BuildCard instead of BuildsTable

### PR DESCRIPTION
**Pull request by katermart**

### CODE REVIEW

File: src/components/builds/BuildCard.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+            <Grid sm="auto" md={3}>
+              <RepositoryNameChipNew repository={build.repository} />
+            </Grid>
+
+            {/* OWNER */}
+            <Grid sm="auto" md={3}>
+              <RepositoryOwnerChipNew repository={build.repository} />
+            </Grid>
+
+            {/* BRANCHE*/}
```
Comment (RDIL) [inline src/components/builds/BuildCard.tsx:133]:
```suggestion
            {/* BRANCH */}
```

File: src/components/builds/BuildCard.tsx [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+        spacing={{ xs: 0.5, sm: 1, md: 3 }}
+        mt={{ sm: 0.5, md: 1.5 }}
+        alignItems={{ xs: 'start', sm: 'center' }}
+        sx={{
+          borderBottom: '1px solid',
+          borderColor: 'divider',
+        }}
+        onClick={e => {
+          const target = e.target as HTMLElement;
+          if (target.closest('a')) return;
```
Comment (fkorotkov) [inline src/components/builds/BuildCard.tsx:122]: Why is this needed?

File: src/components/builds/BuildCard.tsx [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+          navigateBuildHelper(navigate, e, build.id);
+        }}
+        {...rowProps}
+      >
+        {/* LEFT */}
+        <Grid xs={4} sm={3} md={2} mt={{ xs: 1, sm: 0 }}>
```
Comment (fkorotkov) [inline src/components/builds/BuildCard.tsx:128]: 😍

File: src/components/builds/BuildCard.tsx (hunk 2 above)
Comment (katermart) [reply src/components/builds/BuildCard.tsx:122]: To click on the row of the build (with navigation to the build) doesn’t overlap a click on a chip (with navigation somewhere else).
